packages/payment-processor/src/payment/erc20-transferable-receivable.ts

```diff
@@ -0,0 +1,250 @@
+  validateRequest(request, ExtensionTypes.PAYMENT_NETWORK_ID.ERC20_TRANSFERABLE_RECEIVABLE);
+
+  return {
+    data: await encodePayErc20TransferableReceivableRequest(
+      request,
+      signerOrProvider,
+      amount,
+      feeAmountOverride,
+    ),
+    to: getProxyAddress(
+      request,
+      Erc20PaymentNetwork.ERC20TransferableReceivablePaymentDetector.getDeploymentInformation,
+    ),
+    value: 0,
+  };
+}
+
+/**
+ * Encodes the call to pay a request through the ERC20 receivable contract, can be used with a Multisig contract.
+ * @param request request to pay
+ * @param signerOrProvider the Web3 provider, or signer. Defaults to window.ethereum.
+ * @param amount optionally, the amount to pay. Defaults to remaining amount of the request.
+ * @param feeAmountOverride optionally, the fee amount to pay. Defaults to the fee amount of the request.
+ */
+export async function encodePayErc20TransferableReceivableRequest(
+  request: ClientTypes.IRequestData,
+  signerOrProvider: providers.Provider | Signer,
+  amount?: BigNumberish,
+  feeAmountOverride?: BigNumberish,
+): Promise<string> {
+  validateRequest(request, ExtensionTypes.PAYMENT_NETWORK_ID.ERC20_TRANSFERABLE_RECEIVABLE);
+
+  const amountToPay = getAmountToPay(request, amount);
+  const { paymentReference, feeAddress, feeAmount } = getRequestPaymentValues(request);
+  const feeToPay = BigNumber.from(feeAmountOverride || feeAmount || 0);
+
+  const receivableContract = ERC20TransferableReceivable__factory.createInterface();
+
+  const receivableTokenId = await getReceivableTokenIdForRequest(request, signerOrProvider);
+
+  if (receivableTokenId.isZero()) {
+    throw new Error(
+      'The receivable for this request has not been minted yet. Please check with the payee.',
+    );
+  }
+
+  return receivableContract.encodeFunctionData('payOwner', [
+    receivableTokenId, // get tokenId from requestId
+    amountToPay,
+    `0x${paymentReference}`,
+    feeToPay,
+    feeAddress || constants.AddressZero,
+  ]);
+}
+
+/**
+ * Return the EIP-681 format URL with the transaction to pay an ERC20
+ * Warning: this EIP isn't widely used, be sure to test compatibility yourself.
+ *
+ * @param request
+ * @param amount optionally, the amount to pay. Defaults to remaining amount of the request.
+ */
+export function _getErc20TransferableReceivablePaymentUrl(
+  request: ClientTypes.IRequestData,
+  amount?: BigNumberish,
+): string {
+  validateRequest(request, ExtensionTypes.PAYMENT_NETWORK_ID.ERC20_TRANSFERABLE_RECEIVABLE);
+  const contractAddress = getProxyAddress(
+    request,
+    Erc20PaymentNetwork.ERC20TransferableReceivablePaymentDetector.getDeploymentInformation,
+  );
+
+  const reqIdObj = MultiFormat.deserialize(request.requestId);
+  const amountToPay = getAmountToPay(request, amount);
+  const { paymentReference } = getRequestPaymentValues(request);
+  const parameters = `payOwner?uint256=${reqIdObj.value}&uint256=${amountToPay}&bytes=${paymentReference}`;
```

Comment:
I think this should include all of the possible `payOwner` parameters. It's currently missing `feeAmount` and `feeAddress`

```
  function payOwner(
    uint256 receivableTokenId,
    uint256 amount,
    bytes calldata paymentReference,
    uint256 feeAmount,
    address feeAddress
  )
```

Reply:
Discussed offline, we're removing this function